app/presenters/article_presenter.rb

```diff
@@ -1,19 +1,21 @@
 class ArticlePresenter < BasePresenter
-  DOWNLOAD_COMPONENT_IDENTIFIER = 'component_download'.freeze
-
   def download_component
+    return if download_content.blank?
+
     view.render('components/download', content: download_content)
   end
 
-  def download_content
-    DownloadComponent.new(download_block).process
+  def cta_links_component
+    return if cta_links_content.blank?
+
+    view.render('components/cta_links', content: cta_links_content)
```

Comment:
constant?

Reply:
When I was implementing, I moved to a constant. But I thought for a while ... these are usually code from the view inside the presenter. It is just felt awkward to move to a constant a render partial_name to me since on the view I would not do this. I know it is controversial but it felt wrong to me.